priv/rel/files/boot

```diff
@@ -115,8 +115,22 @@ adjust_name() {
     esac
 }
 
+clean_mutable_config_dir() {
+    if [ -n "$RELEASE_MUTABLE_DIR" ] ; then
+        printf "cleaning $GENERATED_CONFIG_DIR ..."
+        rm -rfv "$GENERATED_CONFIG_DIR/*"
+    fi
+}
+
+remove_mutable_config_dir() {
```

Comment:
oh wait - do we really need clean and remove?
i see no need for cleaning, when we have removal. (and i also prefer the latter)